tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -207,13 +207,16 @@ def collect_results(self, sleep_time_secs):
     @cluster(num_nodes=7)
     @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
             broker_type=["leader", "controller"],
+            num_threads=[1, 3],
             sleep_time_secs=[120])
-    def test_broker_type_bounce(self, failure_mode, broker_type, sleep_time_secs):
+    def test_broker_type_bounce(self, failure_mode, broker_type, sleep_time_secs, num_threads):
         """
         Start a smoke test client, then kill one particular broker and ensure data is still received
-        Record if records are delivered. 
+        Record if records are delivered.
+        We also add a single thread stream client to verify whether an reassignment after partition lost
```

Comment:
@guozhangwang The reasoning is logged here.

Reply:
Can you clarify this comment a bit? What does this mean/why should we `verify whether an reassignment after partition lost is still good to proceed` -- isn't an assignment after partitions lost by default not "good to proceed"?

Reply:
I rephrased it a bit.